File: YvetteFB/fizz_buzz.py
for n in range(1, 101):

    if n % 3 == 0 and n % 5 == 0:
        print('fizzbuzz', end=' ')

    elif n % 3 == 0:
        print('fizz', end=' ')

    elif n % 5 == 0:
        print('buzz', end=' ')

    else:
        print(n, end=' ')

File: wambua-iv/fizzbuzz.go
package main

import "fmt"

func FizzBuzz (limit int) {
	for i:=1; i <= limit; i++ {
		if i % 3 == 0 && i % 5 == 0{
			fmt.Println("FizzBuzz")
		} else if i % 3 == 0 {
			fmt.Println("Fizz")
		} else if i % 5 == 0 {
			fmt.Println("Buzz")
		} else {
			fmt.Println(i)
		}
	} 
	
}